packages/components/src/table/TableRow.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, type StylePropType } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import Text from '../text/Text';
+import ExtendedTouchable from '../ExtendedTouchable';
+
+type Props = {|
+  +label: string,
+  +highlightedLabel?: boolean,
+  +renderLabel?: React.Node,
+  +value?: string,
+  +highlightedValue?: boolean,
+  +renderValue?: React.Node,
+  +onPress?: () => void,
+  +style?: StylePropType,
+|};
+
+class TableRow extends React.Component<Props> {
+  renderRow = () => {
```

Comment:
Can you make a component from this method?

Reply:
Yes, done